CovertInfiltration/Src/CovertInfiltration/Classes/XComGameState_CovertActionExpirationManager.uc

```diff
@@ -0,0 +1,118 @@
+//---------------------------------------------------------------------------------------
+//  AUTHOR:  statusNone
+//  PURPOSE: Manager class to keep track of covert infiltration expirations
+//---------------------------------------------------------------------------------------
+//  WOTCStrategyOverhaul Team
+//---------------------------------------------------------------------------------------
+
+class XComGameState_CovertActionExpirationManager extends XComGameState_BaseObject;
+
+struct ExpiringAction
+{
+	var() StateObjectReference	ActionRef;
+	var() TDateTime				Expiration;
+	var() TDateTime				OriginTime;
+};
```

Comment:
* Remove `()` - it isn't needed
* Use just one space between type and field name instead of tabs - tabs look radically different when viewed in various places
* Add `var bool bBlockMonthlyCleanup` to show that the action should not be removed on supply drop (we won't implement this functionality yet, that will be another PR)